### PR TITLE
:pencil: Doc and justfile updates

### PR DESCRIPTION
**Pull request by jefftriplett**
I did some light updates earlier and updated our justfile to start documenting some better workflows.

### CODE REVIEW

**Comment by jefftriplett** [comment]
I'm going to try to spend some time rewriting parts of the docs to make onboarding new users easier. I have had a few people get lost along the way, so I'm going merge and do some follow up.

File: justfile [hunk 1]
```diff
@@ -50,40 +60,122 @@ bootstrap *ARGS:
...
+# --------------------------------------------------
+# Caddy recipes
+# --------------------------------------------------
+
+# Format our Caddyfile
 @caddy-fmt:
     docker-compose run --rm caddy caddy fmt -overwrite /etc/caddy/Caddyfile
 
-# ???
+# Ir our Caddyfile valid?
```
Comment (ryancheley) [inline justfile:79]: @jefftriplett looks like there's a typo here

`Ir our Caddyfile valid?`

Should be

`Is our Caddyfile valid?`

Comment (ryancheley) [reply justfile:79]: This should be fixed in #907

Comment (jefftriplett) [reply justfile:79]: 🤦  good catch. I have a huge stash that I trying not to lose while doing this.

**Pull request merged**